### PR TITLE
Add package.json script names from slack discussion

### PR DESCRIPTION
**Pull request by calebeby**
Slack discussion: https://cloudfour.slack.com/archives/C03ECJ38E/p1611616301002900

I made a new file: `package-json-script-names.md`, but I don't know where it belongs. The `npm` folder has a README but it is about _publishing_ to npm. Open to suggestions

### CODE REVIEW

**Review by emersonthis — APPROVED** [review]
Works for me. Let's get a ✅ from Tyler before merging. Thanks for doing this!

**Comment by gerardo-rodriguez** [comment]
Maybe I'm missing part of the conversation but is there a reason why the contents of the extra file can't be embedded directly in the `testing/README.md` file? Seems a bit redundant and more work to maintain in the long run. I am open to enlightenment. ¯\\\_(ツ)_/¯

I don't know?

**Comment by calebeby** [comment]
@gerardo-rodriguez I am slightly reluctant to because many of the script names are not really testing-related (i.e. linting, typechecking). But the argument could be made that those are sorta like tests

🤷 I do not have a strong opinion on this

File: package-json-script-names.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- `npm run test`: Runs all test runners (Jest and/or Cypress)
+- `npm run test:watch`: Runs all test runners in watch mode
+- `npm run test:jest`: Runs Jest and exits
+- `npm run test:jest:watch`: Runs Jest in watch mode
+- `npm run test:cypress`: Runs Cypress and exits
+- `npm run test:cypress:open`: Open Cypress dashboard
+
+## Linting
+
+- `npm run lint`: Runs all linters and formatters, and fixes any auto-fixable problems
```
Comment (spaceninja) [inline package-json-script-names.md:18]: Worth mentioning that this setup assumes we're running Prettier as part of the ESLint and Stylelint setups? And if you're on a project where you want to keep prettier separate, it'll be `npm run lint:prettier`?

**Review by tylersticka — APPROVED** [review]

File: package-json-script-names.md (hunk 1 above)
Comment (calebeby) [reply package-json-script-names.md:18]: @spaceninja I was trying to be more general by saying "linters and formatters" rather than "eslint and prettier"

**Review by gerardo-rodriguez — APPROVED** [review]

**Pull request merged**